src/body.rs

```diff
@@ -485,10 +485,9 @@ fn guess_ext(path: &Path) -> Option<Mime> {
     match ext {
         Some("html") => Some(mime::HTML),
         Some("js") | Some("mjs") => Some(mime::JAVASCRIPT),
-        Some("json") => Some(mime::JSON),
+        Some("json") | Some("jsonp") => Some(mime::JSON),
```

Comment:
I think jsonp is actually "application/javascript" because it's of the form callback(data) https://blog.abelotech.com/posts/json-jsonp-correct-mime-type/

Reply:
ahhh, gotcha!